service/gcsutils/gcstools/nvidiaPrestartHook.go

```diff
@@ -0,0 +1,71 @@
+package main

```

Comment:
Interesting. I had assumed this tool would be part of the vhd that gets mounted in. Do we want it as a UVM builtin?

Reply:
I don't see a big issue with putting it in the vhd. The only concern would be that if we wanted to say update the lcowNvidiaMountPath we'd have to have LSG update the vhd image.